Core/Graphics/InternalInclude/Babylon/Graphics/DeviceContext.h

```diff
@@ -114,7 +115,7 @@ namespace Babylon::Graphics
         void AddTexture(bgfx::TextureHandle handle, uint16_t width, uint16_t height, bool hasMips, uint16_t numLayers, bgfx::TextureFormat::Enum format);
         void RemoveTexture(bgfx::TextureHandle handle);
         TextureInfo GetTextureInfo(bgfx::TextureHandle handle);
-
+        static inline bx::DefaultAllocator allocator{};
```

Comment:
nit: Instead of exposing the `bx::DefaultAllocator` type, maybe it should better to expose the `bx::AllocatorI` type?

I would suggest keeping this member private and adding an accessor function that returns `bx::AllocatorI&`. Remember that public members should be UpperCamelCase.

Reply:
It makes sense. Fixed.